From 46e5a50e913add1cca9dc96da45e32a3d2fa6e1d Mon Sep 17 00:00:00 2001
From: Ylarod <me@ylarod.cn>
Date: Thu, 20 Jul 2023 10:26:34 +0800
Subject: [PATCH 1/2] Florida: pool-frida

---
 gum/gum.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gum/gum.c b/gum/gum.c
index 58e83a94..1fa2459e 100644
--- a/gum/gum.c
+++ b/gum/gum.c
@@ -302,7 +302,7 @@ gum_init_embedded (void)
   g_log_set_default_handler (gum_on_log_message, NULL);
   gum_do_init ();
 
-  g_set_prgname ("frida");
+  g_set_prgname ("ggbond");
 
 #if defined (HAVE_LINUX) && defined (HAVE_GLIBC)
   gum_libdl_prevent_unload ();
-- 
2.50.1 (Apple Git-155)

