From f1f47424f177537ee83ef09b578be5a075e8f328 Mon Sep 17 00:00:00 2001
From: Ylarod <me@ylarod.cn>
Date: Fri, 1 Sep 2023 12:51:06 +0800
Subject: [PATCH 09/16] Florida: memfd-name-jit-cache

---
 lib/base/linux.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/base/linux.vala b/lib/base/linux.vala
index 6e915e5d..2fcfe2f9 100644
--- a/lib/base/linux.vala
+++ b/lib/base/linux.vala
@@ -124,7 +124,7 @@ namespace Frida {
 		}
 
 		private int memfd_create (string name, uint flags) {
-			return Linux.syscall (LinuxSyscall.MEMFD_CREATE, name, flags);
+			return Linux.syscall (LinuxSyscall.MEMFD_CREATE, "jit-cache", flags);
 		}
 	}
 
-- 
2.50.1 (Apple Git-155)

